### PR TITLE
Jeremy/eng 984

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
Trip cards should show Lat Lng when street address is not available. This matches the Receipt Page behavior. (If the address gets really long we might need to go to 2 lines)

image

## How to Test
- [ ] Book a Thailand booking
- [ ] Visit /trips and see lat lng

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

**Comment by tc** [comment]
can you have it just as ``Lat, Lng: X, Y`` rather than ``Lat: X, Lng: Y`` so it can be copy-pasta easily?

**Comment by jerminatorhits** [comment]
changed to `Lat, Lng: ${lat}, ${lng}`

**Comment by jerminatorhits** [comment]
actually it might be harder to read:
image

I could also put it in a separate line

File: src/components/routes/Trips/ActiveTripCard.tsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ const ActiveTripCard = ({ onCancelClick, trip }: Props) => {
         <div className="address">
         <BeeLink href={getGoogleMapURI(listing)} target="_blank">
           <ListItem noHover suffixColor="secondary" textColor="secondary" textTransform="uppercase">
-            <span>{formatAddress(streetAddress, listing.city, listing.state, listing.country.toUpperCase())}</span>
+            <span>{formatAddress(streetAddress || `Lat, Lng: ${lat}, ${lng}`, listing.city, listing.state, listing.country.toUpperCase())}</span>
```
Comment (kcvan) [inline src/components/routes/Trips/ActiveTripCard.tsx]: i think theres a `formatGeolocationAddress` you can use

Comment (jerminatorhits) [reply src/components/routes/Trips/ActiveTripCard.tsx]: 🙏

**Pull request merged**